project/Whitesource.scala

```diff
@@ -13,14 +13,13 @@ object Whitesource extends AutoPlugin {
     // do not change the value of whitesourceProduct
     whitesourceProduct := "Lightbend Reactive Platform",
     whitesourceAggregateProjectName := {
-       "akka-management-" + (
-         if (isSnapshot.value)
-           if (gitCurrentBranch.value == "master") "master"
-           else "adhoc"
-         else majorMinor(version.value).map(_ + "-current").getOrElse("snapshot"))
-     }
-   )
-
-   def majorMinor(version: String): Option[String] =
-     """\d+\.\d+""".r.findFirstIn(version)
+      val projectName = (moduleName in LocalRootProject).value.replace("-root", "")
+      projectName + "-" + (
+        if (isSnapshot.value)
+          if (gitCurrentBranch.value == "master") "master"
+          else "adhoc"
+        else "stable"
```

Comment:
It is a bit unfortunate this this logic needs to be in every project that uses sbt-whitesource. Would it be possible to move that to the sbt plugin itself?